src/components/Comments.jsx

```diff
@@ -1,27 +1,60 @@
-import React, { useEffect, useState } from 'react'
-import { getAPI } from '../API'
+import React, { useContext, useEffect, useState } from 'react'
+import { deleteAPI, getAPI } from '../API'
+import { UserContext } from '../contexts/User'
 import PostComment from './PostComment'
 
 const Comments = ({reviewID}) => {
     const [renderedComments, setRenderedComments] = useState([])
+    const {currentUser} = useContext(UserContext)
     
+    const handleClick = (event) => {
+        const invisElement = document.getElementById(`deleting-${event.target.value}`)
+        const buttonElement = document.getElementById(`button-${event.target.value}`)
+
+        invisElement.style.display = "block"
+        buttonElement.style.display = "none"
+
+        deleteAPI(event.target.value)
+            .then(() => {
+                const commentElement = document.getElementById(`comment-${event.target.value}`)
+                
+                commentElement.style.display = "none"
+            })
+            .catch(() => {
+                const failureElement = document.getElementById(`failure-${event.target.value}`)
+
+                failureElement.style.display = "block"
+                invisElement.style.display = "none"
+                buttonElement.style.display = "block"
+            })
+    }
+
+    const renderCommentsArr = (comments) => {
```

Comment:
Can use ternary here to return different part in the final return. But this way is also OK